.github/workflows/java.yml

```diff
@@ -1,4 +1,4 @@
-# Copyright 2024 New Relic, Inc.
+# Copyright 2010 New Relic, Inc.
```

Comment:
We're going back in time? But also, do we really need to add a license to workflows?

Reply:
I saw that, we're supposed to replace that with just the SPDX identifier (I think that's the acronym).

But historically yes, we add that to all our workflows since they sometimes contain non-trivial code.

Reply:
Intriguing.